### PR TITLE
Deprecate duplicate functions from utils.

### LINKED ISSUE #245: Remove utils functions existing in codemirror.
These [two](https://github.com/jupyter/notebook/blob/master/notebook/static/base/js/utils.js#L492-L528) utils functions seem to exist [in codemirror](https://github.com/codemirror/CodeMirror/blob/master/lib/codemirror.js#L7572-L7590) 

I guess they are more reliable.

### PR DESCRIPTION
**Pull request by Carreau**
Closes #245

### CODE REVIEW

**Comment by ellisonbg** [comment]
Sad Travis :(

**Comment by Carreau** [comment]
> Sad Travis :(

You know you can head to the travis page and restart just the failing test ?

File: notebook/static/base/js/utils.js [hunk 1]
```diff
@@ -500,41 +500,13 @@ define([
...
-         */
-        if (!cursor) {
-            cursor = cm.getCursor();
-        }
-        var cursor_pos = cursor.ch;
-        for (var i = 0; i < cursor.line; i++) {
-            cursor_pos += cm.getLine(i).length + 1;
-        }
-        return cursor_pos;
+        console.warn('utils.to_absolute_cursor_pos(cm, pos) is deprecated us cm.indexFromPos(cursor)');
```
Comment (damianavila) [inline notebook/static/base/js/utils.js]: ... ` is deprecated us`, you want to put `use`? `by`? I don't get the phrase written as is...

below it is the same... thanks!

Comment (Carreau) [inline notebook/static/base/js/utils.js]: Better/

Comment (damianavila) [inline notebook/static/base/js/utils.js]: Perfect! thanks!

**Pull request merged**